picoquic_crypto/ptls_fusion_backend.c

```diff
@@ -0,0 +1,23 @@
+/*
+* Copyright (c) 2020, Private Octopus, Inc., The PicoQuic Authors.
```

Comment:
I started working on integration of fusion -- the first step was done inside the Picotls project, to add the fusion code to a benchmark program. This requires changes in the generic AEAD implementation in tls_api.c. I think those changes need to be done before we start splitting some of the code in a crypto library. I will push a PR for that soon.

Reply:
Great, I am starting with a basic skeleton of the crypto api as a suggestion, It then can be changed at a later state. Also be aware that I will force push this branch